docs/api/1.x/_details-patch-object.rst

```diff
@@ -60,3 +53,38 @@ See :ref:`api-permissions-payload`.
 .. ----------------
 
 .. If a read-only field is modified, a |status-400| error is returned.
+
+JSON Patch Operations
+---------------------
+
+`JSON-Patch <https://tools.ietf.org/html/rfc6902>`_ is a way to define a sequence
+of operations to be applied on a JSON object.
+
+It's possible to use ``JSON-Patch`` notation setting ``Content-Type: application/jso-patch+json``.
+
+When using this `patch` method, the request should be defined as a list of operations, for example:
```

Comment:
When using this request header, the body should contain a list of operations. For example: